lib/ui/view/forgot_password_email/forgot_password_email_view.dart

```diff
@@ -57,13 +66,30 @@ class ForgotPasswordEmailView extends StatelessWidget {
                 SizedBox(
                   height: 49.0,
                 ),
-                CustomTextField(
-                  keyboardType: TextInputType.emailAddress,
-                  inputAction: TextInputAction.done,
-                  autoCorrect: true,
-                  obscureText: false,
-                  labelText: 'Email',
-                  hintText: 'Name@gmail.com',
+                Form(
+                  // autovalidateMode: AutovalidateMode.onUserInteraction,
+                  key: validateKey,
+                  child: CustomTextField(
+                    controller: emailController,
+                    //autovalidateMode: AutovalidateMode.onUserInteraction,
+                    validator: (val) {
```

Comment:
Move all validation to a mixing called `util/validators.dart` then add as a mixing in the viewmodel, then you can simply call `model.validateEmail` or `model.validatePhoneNumber` and use this mixing in any viewmodel

Reply:
changes made